region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -707,6 +708,41 @@
       "farmCycleDrops": [{"enemy": "Covern", "count": 1}],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Pause Abuse Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          {"and": [
+            "h_CrystalFlashForReserveEnergy",
+            "canTrickyDodgeEnemies"
+          ]},
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"enemyKill": {"enemies": [["Bull", "Bull"]], "excludedWeapons": ["PowerBomb"]}},
+            {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+          ]}
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 14, "openEnd": 0}},
+          {"and": [
+            "f_DefeatedPhantoon",
+            {"canShineCharge": {"usedTiles": 20, "openEnd": 2}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
```

Comment:
- The bulls didn't feel easier to avoid than just using the spikes.  You have to keep them away long enough to shortcharge & keep them close enough to interrupt on.  I was messing up on them running into me during the windup; they kept missing.
- The Robot lasers only seemed to shoot 1 usable shot, and I found that to be too unreliable.

So imo it should require the pause abuse interrupt.